### PR TITLE
HDDS-4899. Add SCM HA to Chaos tests.

### PR DESCRIPTION
**Pull request by mukul1987**
## What changes were proposed in this pull request?
This jira adds SCM HA to Chaos tests

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4899

## How was this patch tested?
Added a new chaos test

### CODE REVIEW

**Comment by GlenGeng** [comment]
Please hold on the merge util we finish merging master back to 2823. Thanks!

File: hadoop-ozone/fault-injection-test/mini-chaos-tests/src/test/java/org/apache/hadoop/ozone/MiniOzoneChaosCluster.java [hunk 1]
```diff
@@ -382,4 +408,45 @@ public void shutdownHddsDatanode(DatanodeDetails dn) throws IOException {
...
+  public Set<StorageContainerManager> scmToFail() {
+    int numNodesToFail = getNumberOfScmToFail();
+    if (failedScmSet.size() >= numStorageContainerManagers/2) {
+      return Collections.emptySet();
+    }
+
+    int numOms = getStorageContainerManagersList().size();
+    Set<StorageContainerManager> scms = new HashSet<>();
+    for (int i = 0; i < numNodesToFail; i++) {
+      int failedNodeIndex = FailureManager.getBoundedRandomIndex(numOms);
```
Comment (bshashikant) [inline hadoop-ozone/fault-injection-test/mini-chaos-tests/src/test/java/org/apache/hadoop/ozone/MiniOzoneChaosCluster.java:426]: numOms--> numSCMs

Comment (mukul1987) [reply hadoop-ozone/fault-injection-test/mini-chaos-tests/src/test/java/org/apache/hadoop/ozone/MiniOzoneChaosCluster.java]: Done

**Review by bshashikant — APPROVED** [review]

**Comment by bshashikant** [comment]
Thanks @mukul1987 for the contribution.

**Pull request merged**